src/FSharpVSPowerTools.Logic/QuickInfoMargin.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<UserControl xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:sys="clr-namespace:System;assembly=mscorlib" 
+             xmlns:vs="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.11.0" 
+             xmlns:pt="clr-namespace:FSharpVSPowerTools.QuickInfo"
+             mc:Ignorable="d" 
+             Height="24"
+             d:DesignHeight="25" d:DesignWidth="600">
+    <TextBox IsEnabled="True" IsReadOnly="True" Text="{Binding QuickInfo, Mode=OneWay}" Background="{DynamicResource {x:Static SystemColors.ControlBrushKey}}" />
```

Comment:
Is there any XAML trick to make double click on the text box a select all command? Currently I have to press Ctrl + A to select all before copy the text.